mpt/impl/imap-mailbox/external-james/pom.xml

```diff
@@ -68,4 +68,35 @@
             <artifactId>commons-lang3</artifactId>
         </dependency>
     </dependencies>
+
+    <profiles>
+        <profile>
+            <id>dev</id>
```

Comment:
this profile name is strange, no? If you are in dev, you want to be able to play easily all the tests, no?

Reply:
ok, I will merge this then